flink-runtime/src/main/scala/org/apache/flink/runtime/minicluster/FlinkMiniCluster.scala

```diff
@@ -51,10 +51,13 @@ import scala.concurrent.{ExecutionContext, Future, Await}
 abstract class FlinkMiniCluster(
     val userConfiguration: Configuration,
     val singleActorSystem: Boolean,
+    val synchronousDispatcher: Boolean,
     val streamingMode: StreamingMode) {
 
-  def this(userConfiguration: Configuration, singleActorSystem: Boolean) 
-         = this(userConfiguration, singleActorSystem, StreamingMode.BATCH_ONLY)
+  def this(userConfiguration: Configuration,
+           singleActorSystem: Boolean,
+           synchronousDispatcher: Boolean)
+       = this(userConfiguration, singleActorSystem, synchronousDispatcher, StreamingMode.BATCH_ONLY)
```

Comment:
Good catch. I noticed that as well and already changed that before I saw your answer. Now only the `TestingCluster` is affected by the changes.